x-pack/test/fleet_api_integration/apis/epm/install_integration_in_multiple_spaces.ts

```diff
@@ -130,5 +134,41 @@ export default function (providerContext: FtrProviderContext) {
       const spaceTag = await getTag(`fleet-pkg-${pkgName}-fleet_test_space`, testSpaceId);
       expect(spaceTag).not.equal(undefined);
     });
+
+    it('should keep assets in space when format version is bumped', async () => {
+      const nginxPkgName = 'nginx';
+      const nginxPkgVersion = '1.17.0';
+
+      const installResponse = await installPackageInSpace(
+        nginxPkgName,
+        nginxPkgVersion,
+        testSpaceId
+      );
+
+      const firstAsset = installResponse.body.items.find((item: any) => item.type === 'dashboard');
+
+      // Bump format version directly on installation saved object, then call setup to trigger a reinstall
+      await es.update({
+        index: INGEST_SAVED_OBJECT_INDEX,
+        id: `${PACKAGES_SAVED_OBJECT_TYPE}:${nginxPkgName}`,
+        body: {
+          doc: {
+            [PACKAGES_SAVED_OBJECT_TYPE]: {
+              install_format_schema_version: 999,
```

Comment:
Should this be a small number like 1? we are looking for `install_format_schema_version < ${FLEET_INSTALL_FORMAT_VERSION}` in the setup